crates/sdk/tests/idiom.rs

```diff
@@ -335,6 +335,14 @@ async fn idiom_recursion_record_links() -> Result<(), Error> {
 		planet:earth.{3}(.contains).name;
 		planet:earth.{4}(.contains).name;
 
+		planet:earth.{2+path}(.contains);
```

Comment:
Could we also add various invalid path syntaxes and check that error message?

Reply:
Added some extra tests in 160b82d